wicket/src/ui/panes/update.rs

```diff
@@ -42,14 +49,22 @@ pub struct UpdatePane {
     /// capture all state.
     tree_state: TreeState,
     items: Vec<TreeItem<'static>>,
+
+    // Per-component update state that isn't serializable.
```

Comment:
This is going to prevent recording snapshots from ever working if we exceed our number of max events.  This is why @jgallagher had to increase the buffer size so much the other day, which may not be feasible for production workloads. We don't really know how long wicket is going to be open or how many events to expect and we don't have unlimited memory.

In effect adding more state here prevents us from being able to use this in production. If you notice above, there's a comment about moving `tree_state` into global `State`. I'd really prefer we don't add more state here. Is there a way that you can make this serializable and add it to the global `State`? 

Really, there shouldn't be any type of state maintained here. This likely also means moving the `Rect`s out into `State`. We have to do this if we ever support resize events in recordings because we will lose the resize events when we take a snapshot.

Reply:
In a meeting we decided to not block on this. @andrewjstone filed https://github.com/oxidecomputer/omicron/issues/3070 to track this.